cms/static/js/spec/views/paging_spec.js

```diff
@@ -380,193 +384,5 @@ define([ "jquery", "common/js/spec_helpers/ajax_helpers", "URI",
                     });
                 });
             });
-
-            describe("PagingFooter", function () {
```

Comment:
I think it would make sense to keep some minimal footer tests here. For example, the asset tests would all now pass if the footer was removed entirely, which doesn't seem right. Maybe just verify that it is possible to use the next and previous buttons.